Collectors/wlcg_converter.py

```diff
@@ -183,6 +184,22 @@ def Convert(source_record):
     
     return to_return
 
+# check the path on TPC     
+def tpcPathCheckWLCG(pathCheck):
+    try:
+        parts =  re.split('(.*):\/\/([\w\-\.]+)(\:*[0-9]*)(\/+)(.*)',pathCheck)
+    
+        path = parts[5]
```

Comment:
Just send this line (192) to after the "if" to prevent and out of index error when the "if" statement is true